### PR TITLE
{Release} HotFix: Release 2.26.1

### PR DESCRIPTION
**Pull request by zhoxing-ms**
**Description**

Related PR
1. https://github.com/Azure/azure-cli/pull/18853
2. https://github.com/Azure/azure-cli/pull/18795
3. https://github.com/Azure/azure-cli/pull/18811
4. https://github.com/Azure/azure-cli/pull/18730

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

**Review by 00Kai0 — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,23 @@
...
+
+**AKS**
+
+* Hotfix: `az aks create`: Fix the issue that `assign-kubelet-identity` option can't work
+
+**Storage**
+
+* Hotfix: Fix issue caused by jwt upgrade.
+* Hotfix: `az storage fs directory download`: Fix the issue with `--sas-token` to generate valid sas url
+* Hotfix: `az storage blob copy start`: Fix the issue in copy from different account
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:21]: Shouldn't we include the PR number the release history?

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,23 @@
...
+
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:22]: Shouldn't we include the PR number in the release history?

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:21]: Words like `bug`, `issue` are too general and cannot explicitly reflect the item that is fixed.

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,23 @@
 Release History
 ===============
 
+2.26.1
+++++++
+
+**ACR**
+
+* Hotfix: `az acr build\connected-registry\pack\run\scope-map`: Fix the compatibility bug caused by SDK upgrade
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:11]: Shouldn't we include the PR number in the release history?

Comment (zhoxing-ms) [reply src/azure-cli/HISTORY.rst]: Updated

**Review by Juliehzl — APPROVED** [review]

File: src/azure-cli/HISTORY.rst (hunk 3 above)
Comment (Abhi-Balijepalli) [reply src/azure-cli/HISTORY.rst]: I tried running that but ended up with this error: **'buildconnected-registrypackrunscope-map' is misspelled or not recognized by the system.**

**Pull request merged**